### PR TITLE
Ar 2 iu branding, AR-9 update privacy policy

### PR DESCRIPTION
**Pull request by dlim87**
### Adds IU branded header and footer, updated privacy policy link in footer
Closes:

https://bugs.dlib.indiana.edu/browse/AR-2

https://bugs.dlib.indiana.edu/browse/AR-9

- Adds IU branded Header
- Adds IU branded Footer
- Privacy policy in footer links to library's privacy policy

### Screenshots and video:
Full size:
Image 2020-03-19 at 12 37 11 PM

Tablet (IU logo is flush with the rest of the header, arclight header is suppressed):
Image 2020-03-19 at 12 37 52 PM

Mobile ("Indiana Univeristy" becomes "IU" in header, centennial footer takes up whole screen, footer elements are centered)
Image 2020-03-19 at 12 38 09 PM

Video: 
Screen Recording 2020-03-19 at 12 41 PM

### CODE REVIEW

File: app/views/shared/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!--imported from https://assets.iu.edu/brand/3.2.x/footer.html with modifications for image-->
+<footer id="footer" role="contentinfo" itemscope="itemscope" itemtype="http://schema.org/CollegeOrUniversity">
+  <div class="row pad">
+
+    <p class="tagline bicentennial celebration"><a href="https://200.iu.edu/"><svg width="223" height="25" viewBox="0 0 223 25" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (dewey4iv) [inline app/views/shared/_footer.html.erb]:
```suggestion
    <p class="tagline bicentennial celebration"><a href="https://200.iu.edu/">
        <svg width="223" height="25" viewBox="0 0 223 25" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment (dewey4iv) [inline app/views/shared/_footer.html.erb]: Little nit-picky... but the spacing here is a bit off.

File: app/views/shared/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    </p>
+    <div class="footer-links">
+      <p class="signature">
+        <a href="https://www.iu.edu/" class="signature-link signature-img">
+          <%= image_tag('iu-sig-formal.svg') %>
+        </a>
+      </p>
+
+      <p class="copyright">
+  <!--      TODO: update privacy policy link-->
```
Comment (labradford) [inline app/views/shared/_footer.html.erb]: is this still at todo?

Comment (dlim87) [reply app/views/shared/_footer.html.erb]: no, good catch

Comment (dlim87) [reply app/views/shared/_footer.html.erb]:
```suggestion
```

File: app/views/shared/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<!--imported from https://assets.iu.edu/brand/3.2.x/footer.html with modifications for image-->
+<footer id="footer" role="contentinfo" itemscope="itemscope" itemtype="http://schema.org/CollegeOrUniversity">
+  <div class="row pad">
+
```
Comment (dlim87) [inline app/views/shared/_footer.html.erb]:
```suggestion
```

**Pull request merged**